programs/uxd/src/events.rs

```diff
@@ -28,6 +28,21 @@ pub struct SetRedeemableGlobalSupplyCapEvent {
     pub redeemable_global_supply_cap: u128,
 }
 
+/// Event called in [instructions::edit_controller::handler].
+#[event]
+pub struct SetDepositoriesWeightBps {
+    /// The controller version.
+    #[index]
+    pub version: u8,
+    /// The controller.
+    #[index]
+    pub controller: Pubkey,
+    /// The new weights
+    pub identity_depository_weight_bps: u16,
+    pub mercurial_vault_depository_0_weight_bps: u16,
+    pub credix_lp_depository_0_weight_bps: u16,
```

Comment:
note: I am putting a number on those because this weight is specifically assigned to the:
 `registered_credix_lp_depositories[0]` referenced in the controller